experiments/ClimaEarth/leaderboard/leaderboard.jl

```diff
@@ -153,11 +160,151 @@ function compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
     CairoMakie.save(joinpath(leaderboard_base_path, "bias_leaderboard.png"), fig_leaderboard)
 end
 
+"""
+    compute_pfull_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+
+Plot the biases and a leaderboard of various variables in pressure coordinates.
+
+The paramter `leaderboard_base_path` is the path to save the leaderboards and bias plots and
+`diagnostics_folder_path` is the path to the simulation data.
+"""
+function compute_pfull_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+    @info "Error against observations for variables in pressure coordinates"
+
+    sim_var_pfull_dict = get_sim_var_in_pfull_dict(diagnostics_folder_path)
+    obs_var_pfull_dict = get_obs_var_in_pfull_dict()
+
+    # Print dates for debugging
+    _, get_var = first(sim_var_pfull_dict)
+    var = get_var()
+    output_dates = Dates.DateTime(var.attributes["start_date"]) .+ Dates.Second.(ClimaAnalysis.times(var))
+    @info "Working with dates:"
+    @info output_dates
+
+    # Set up dict for storing simulation and observational data after processing
+    sim_obs_comparsion_dict = Dict()
+
+    for short_name in keys(sim_var_pfull_dict)
+        sim_var = sim_var_pfull_dict[short_name]()
+        obs_var = obs_var_pfull_dict[short_name](sim_var.attributes["start_date"])
+        # Remove first spin_up_months from simulation
+        spin_up_months = 6
+        spinup_cutoff = spin_up_months * 31 * 86400.0
+        ClimaAnalysis.times(sim_var)[end] >= spinup_cutoff &&
+            (sim_var = ClimaAnalysis.window(sim_var, "time", left = spinup_cutoff))
+
+        # Restrain the pressure levels so we can resample
+        min_pfull = ClimaAnalysis.pressures(obs_var)[1]
+        sim_pressures = ClimaAnalysis.pressures(sim_var)
+        greater_than_min_pfull_idx = findfirst(x -> >(x, min_pfull), sim_pressures)
```

Comment:
I don't think we use infix binary operators all that much, I'd suggest writing
```suggestion
        greater_than_min_pfull_idx = findfirst(x -> x > min_pfull, sim_pressures)
```
for clarity